crates/editor/src/element.rs

```diff
@@ -2596,6 +2598,60 @@ impl EditorElement {
         }
     }
 
+    #[allow(clippy::too_many_arguments)]
+    fn layout_signature_help(
+        &self,
+        hitbox: &Hitbox,
+        content_origin: gpui::Point<Pixels>,
+        scroll_pixel_position: gpui::Point<Pixels>,
+        display_point: Option<DisplayPoint>,
+        start_row: DisplayRow,
+        line_layouts: &[LineWithInvisibles],
+        line_height: Pixels,
+        em_width: Pixels,
+        cx: &mut WindowContext,
+    ) {
+        let Some(display_point) = display_point else {
+            return;
+        };
+
+        let cursor_row_layout = &line_layouts[display_point.row().minus(start_row) as usize];
+        let start_x = cursor_row_layout.x_for_index(display_point.column() as usize)
+            - scroll_pixel_position.x
+            + content_origin.x;
+        let start_y =
+            (display_point.row().next_row().as_f32() + 1.0) * line_height - scroll_pixel_position.y;
```

Comment:
`.next_row().as_f32() + 1.0` is two rows forward, why 2 specifically?

Reply:
This is leftover from debugging the display position. I apologize for that.
e29a6c1af